lamindb/models/schema.py

```diff
@@ -213,18 +294,14 @@ class Meta(Record.Meta, TracksRun.Meta, TracksUpdates.Meta):
     For a composite schema, the hash of hashes.
     """
     minimal_set: bool = BooleanField(default=True, db_index=True, editable=False)
```

Comment:
Please make a comment here that this is deprecated.

And add to the list of future migrations:

- https://github.com/laminlabs/lamindb/issues/2604